controllers/C_Document.class.php

```diff
@@ -198,7 +198,7 @@ function upload_action_process()
                         $filetext = $this->cryptoGen->decryptStandard($filetext, $passphrase);
                         if ($filetext === false) {
                             error_log("OpenEMR Error: Unable to decrypt a document since decryption failed.");
-                            $filetext = "";
+                            errorLogEscape($filetext = "");
```

Comment:
this line should not be changed since it is not related to the above error_log call

Reply:
Okay, reverted :) !